telegram/message.py

```diff
@@ -880,6 +880,22 @@ def delete(self, *args, **kwargs):
         return self.bot.delete_message(
             chat_id=self.chat_id, message_id=self.message_id, *args, **kwargs)
 
+    def stop(self, *args, **kwargs):
```

Comment:
`def stop_poll`?

Reply:
No, thats already defined.

Reply:
Where?

Reply:
grr, I dont see it. I was so sure I got a warning thrown at me when I wrote down stop_poll